salt/matchers/compound_pillar_exact_match.py

```diff
@@ -14,7 +12,7 @@
 log = logging.getLogger(__name__)
 
 
-def mmatch(expr, delimiter, greedy, opts=None):
+def mmatch(expr, delimiter, greedy, opts=None, minion_id=None):
```

Comment:
Why are we passing `minion_id` here but not using it?

Reply:
Because all of them now need to accept it as a keyword argument?

Reply:
The `compound_match` function uses all the other match functions and passes it along as an argument, so we need to make sure it's a valid argument for all functions.

Reply:
Actually, I don't think this is necessary? We don't actually pass it along?

Reply:
Ie, `compound_match` needs the `minion_id` kwarg, but this is `compound_pillar_exact_match`